fastlane/lib/fastlane/setup/setup.rb

```diff
@@ -47,8 +47,7 @@ def is_react_native?
     end
 
     def show_analytics
-      UI.message("fastlane will send the number of errors for each action to")
-      UI.message("https://github.com/fastlane/enhancer to detect integration issues")
+      UI.message("fastlane will collect the number of errors for each action to detect integration issues")
```

Comment:
Should we add a link to the docs here, and information about adding `opt_out_usage`? As a developer I think it makes sense, that when I see the message, that I also have a way to disable it now, without having to run _fastlane_ first.